routes/api.php

```diff
@@ -165,7 +164,7 @@
 );
 ## this is no good. StreamController::index does not process book_id/chapter/verse_start/verse_end
 Route::name('v4_media_stream')->get(
-    'bible/filesets/{fileset_id}/{book_id}-{chapter}-{verse_start?}-{verse_end?}/playlist.m3u8',
```

Comment:
@vichugofsl this seems to imply that the download endpoint is calling the v4_media_stream endpoint, since DBP-353 has to do with downloads. Am I understanding you correctly?

Reply:
@bradflood, yes, you are right. It is calling the next line:
https://github.com/faithcomesbyhearing/dbp/blob/develop/app/Traits/BibleFileSetsTrait.php#L266

The above happens when the fileset has Multiple Mp3 Chapters.

Reply:
I don't want to remove the restriction that chapter is required for general (not download) calls.  Please evaluate options that keep restriction in place for general calls

Reply:
> I don't want to remove the restriction that chapter is required for general (not download) calls. Please evaluate options that keep restriction in place for general calls

Ok, I understand. Let me check it.

Reply:
@bradflood, It has been removed. The chapter parameter is still mandatory to create the route